### PR TITLE
Fix typos and description consistency for naming rules

### PR DESCRIPTION
**Pull request by ahmdaa**
This fixes typos in descriptions for naming rules and makes them consistent.

### CODE REVIEW

File: detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/VariableNaming.kt [hunk 1]
```diff
@@ -23,7 +23,7 @@ import org.jetbrains.kotlin.resolve.calls.util.isSingleUnderscore
 @Alias("PropertyName")
 class VariableNaming(config: Config) : Rule(
     config,
-    "Variable names should follow the naming convention set in the projects configuration."
+    "Variable names should follow the naming convention set in the project's configuration."
```
Comment (cortinico) [inline detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/VariableNaming.kt:26]: I'm not sure using the genitive here is correct. I literally never saw "project's configuration" in my career, but maybe I'm wrong. @3flex are you a native? If yes could you proofread this?

Comment (ahmdaa) [reply detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/VariableNaming.kt:26]: I'm assuming we are referring to a single project here, right? Can also say "project configuration". 🤔

Comment (3flex) [reply detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/VariableNaming.kt:26]: > are you a native?

I am a native English speaker. "project's configuration" looks correct to me.

Comment (ahmdaa) [reply detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/VariableNaming.kt:26]: > Variable names should follow the naming convention set in detekt's configuration

Was thinking this makes more sense as well! Let me change it.

**Review by BraisGabin — APPROVED** [review]

**Comment by BraisGabin** [comment]
Thank you for the improvements!!

**Review by cortinico — APPROVED** [review]

**Pull request merged**